overlay/tensorflow_validation/validation.py

```diff
@@ -1,46 +1,40 @@
+import os
+
 import tensorflow as tf
 import pandas as pd
 from logging import info, error
 from sklearn.preprocessing import MinMaxScaler
 from concurrent.futures import ThreadPoolExecutor, as_completed
 
-from overlay.validation_pb2 import ValidationMetric, MongoReadConfig, ValidationJobRequest, BudgetType, StaticBudget
+from overlay.validation_pb2 import ValidationMetric, ValidationJobRequest, BudgetType, StaticBudget
 from overlay.db.querier import Querier
-from overlay.constants import DB_HOST, DB_PORT, DB_NAME, MODELS_DIR
+from overlay.constants import MODELS_DIR
 
 
 class TensorflowValidator:
 
     def __init__(self, request: ValidationJobRequest):
-        self.job_id = request.id
-        self.model_type = request.model_category
-        self.mongo_host = request.mongo_host
-        self.mongo_port = request.mongo_port
-        self.read_config = request.read_config
-        self.database = request.database
-        self.collection = request.collection
-        self.feature_fields = request.feature_fields
-        self.label_field = request.label_field
-        self.validation_metric = request.validation_metric
-        self.normalize = request.normalize_inputs
-        self.validation_budget = request.validation_budget
+        self.request: ValidationJobRequest = request
 
     def load_tf_model(self, verbose=False):
         # Load Tensorflow model from disk
-        model_path = f"{MODELS_DIR}/{self.job_id}"
+        model_path = f"{MODELS_DIR}/{self.request.id}/"
+        first_entry = os.listdir(model_path)[0]
```

Comment:
Could be either the directory containing the `SavedModel` format, or a `.h5` HDF5 model file